src/Client/MultiplexedConnections.cpp

```diff
@@ -141,13 +141,12 @@ void MultiplexedConnections::sendQuery(
             modified_settings.group_by_two_level_threshold = 0;
             modified_settings.group_by_two_level_threshold_bytes = 0;
         }
+    }
 
-        if (replica_info)
-        {
-            client_info.collaborate_with_initiator = true;
-            client_info.count_participating_replicas = replica_info->all_replicas_count;
-            client_info.number_of_current_replica = replica_info->number_of_current_replica;
-        }
+    if (replica_info)
+    {
+        client_info.collaborate_with_initiator = true;
```

Comment:
why it is still needed in client_info after we removed it from replica_info?

Reply:
It's the current way to understand for a remote node that it should act as follower in context of a query with PR. `client_info.collaborate_with_initiator` is set on the initiator, sent to followers as part of interserver protocol and checked on followers in `Context::canUseParallelReplicasOnFollower()`